### PR TITLE
Add Create Aptos Dapp section to get started -> developer setup

### PR DESCRIPTION
**Pull request by igor-p**
### Description
Adds a `Create Aptos Dapp` section to the [Get Started | Developer Setup](https://aptos.dev/en/build/get-started/developer-setup) page.

image

### Checklist

- Do all Lints pass?
  - [ ] Have you ran `pnpm spellcheck`?
  - [x] Have you ran `pnpm fmt`?
  - [x] Have you ran `pnpm lint`?

### CODE REVIEW

**Review by hariria — COMMENTED** [review]
I think there's a large unintentional lock file change?

**Comment by hariria** [comment]
Also could you add @0xmaayan as a reviewer

File: apps/nextra/utils/generated/localeMap.ts
Comment (igor-p) [inline apps/nextra/utils/generated/localeMap.ts:1]: These changes appeared after I ran `npx turbo run build --filter={apps/nextra}...`. Figure it's ok to keep them

**Comment by igor-p** [comment]
> I think there's a large unintentional lock file change?

Whoops, missed that. Fixed.

**Comment by 0xmaayan** [comment]
@igor-p Looks great! Thank you for working on it! 

Since we're already here, would you mind updating the CLI gif/video to the one we use in Aptos Learn?  😄 
https://learn.aptoslabs.com/en/dapp-templates

**Review by 0xmaayan — APPROVED** [review]

**Pull request merged**